From f5e8f9f77b9944255317650a26eaef04a43a7b44 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Wed, 3 Nov 2021 22:45:12 -0700
Subject: [PATCH 121/233] [FIO toup] apalis-imx8: allow dropping CONFIG_CMD_USB
 requirement

When CONFIG_DISTRO_DEFAULTS is not defined, don't include
config_distro_bootcmd.h. This will break the build based
on the stock defines:
        func(MMC, mmc, 1) \
        func(MMC, mmc, 2) \
        func(MMC, mmc, 0) \
        func(USB, usb, 0)

Signed-off-by: Michael Scott <mike@foundries.io>
---
 include/configs/apalis-imx8.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/configs/apalis-imx8.h b/include/configs/apalis-imx8.h
index 9bb42a7042a..db17149e251 100644
--- a/include/configs/apalis-imx8.h
+++ b/include/configs/apalis-imx8.h
@@ -59,13 +59,17 @@
 	"m4boot_0=run loadm4image_0; dcache flush; bootaux ${loadaddr} 0\0" \
 	"m4boot_1=run loadm4image_1; dcache flush; bootaux ${loadaddr} 1\0" \
 
+#ifdef CONFIG_DISTRO_DEFAULTS
 /* Enable Distro Boot */
 #define BOOT_TARGET_DEVICES(func) \
 	func(MMC, mmc, 1) \
 	func(MMC, mmc, 2) \
 	func(MMC, mmc, 0) \
 	func(USB, usb, 0)
 #include <config_distro_bootcmd.h>
+#else
+#define BOOTENV ""
+#endif
 
 /* Initial environment variables */
 #define CONFIG_EXTRA_ENV_SETTINGS \
-- 
2.25.1

